CMakeLists.txt

```diff
@@ -26,6 +26,21 @@ option(USE_TSAN "Enable ThreadSanitizer checks" OFF)
 option(USE_MSAN "Enable MemorySanitizer checks" OFF)
 option(USE_VALGRIND "Enable Valgrind instrumentation" OFF)
 
+set(buildTypes Release Debug RelWithDebInfo MinSizeRel)
```

Comment:
I'm not sure if this is a convention or not, but I think it should be `build_types`

Reply:
Maybe `BUILD_TYPES`?

Reply:
Done

Reply:
I proposed lowercase because it's more of a local variable, but I'm not sure... If you'd like to make it more global (and hence uppercase) I'd re-name it to something like `KNOWN_BUILD_TYPES` or something

Reply:
@lukaszstolarczuk  Done